flow-server/src/main/java/com/vaadin/flow/server/frontend/FrontendTools.java

```diff
@@ -796,6 +792,20 @@ public FrontendVersion getNpmVersion() throws UnknownVersionException {
         return FrontendUtils.getVersion("npm", npmVersionCommand);
     }
 
+    public String getNpmPackageExecutable(String packageName, String binName,
+            File cwd) throws CommandExecutionException {
+        var script = """
+                var jsonPath = require.resolve('%packageName%/package.json');
+                var json = require(jsonPath);
+                console.log(path.resolve(path.dirname(jsonPath), json.bin['%binName%']));
+                """
+                .replace("%packageName%", packageName)
+                .replace("%binName%", binName);
```

Comment:
nit: could use `String.formatted` instead of replace
```suggestion
        var script = """
                var jsonPath = require.resolve('%s/package.json');
                var json = require(jsonPath);
                console.log(path.resolve(path.dirname(jsonPath), json.bin['%s']));
                """.formatted(packageName, binName);
```

Reply:
Not a blocker anyway